Mathematica.java

```diff
@@ -2,4 +2,33 @@
     public static int sum(int a, int b){
         return a + b;
     }
+    public static int power (int a, int b) {
+        int result = a;
+        for (int i = 1; i <= b; i++){
+            result *= a;
+        }
+        return result;
+    }
+    public static int subtract(int a, int b) {
+        return a - b;
+    }
+    public static int multiplication(int a, int b){
+        return a * b;
+    }
+    public static int division(int a, int b){
+        return a / b;
+    }
+    public static int Abs(int x){
+        if (x > 0)
+            return x;
+        else
+            return x * -1;
+    }
```

Comment:
What does `factors(int x)` actually do? Just print the factors of an integer? It would be more useful to return the factors and not just print them.